vividus-engine/src/main/java/org/vividus/expression/NormalizingArgumentsUtils.java

```diff
@@ -18,10 +18,26 @@
 
 import org.apache.commons.lang3.StringUtils;
 
-interface NormalizingArguments
+final class NormalizingArgumentsUtils
 {
-    default String normalize(String argument)
+    private NormalizingArgumentsUtils()
     {
-        return StringUtils.replace(argument.trim(), "\\,", ",");
+    }
+
+    static String normalize(String argument)
+    {
+        if (StringUtils.isNotBlank(argument))
+        {
+            String trimmed = argument.trim();
+            String wrapToken = "\"\"\"";
+            boolean isQuoteWrapped = trimmed.startsWith(wrapToken) && trimmed.endsWith(wrapToken);
```

Comment:
```suggestion
            boolean wrappedInQuotes = trimmed.startsWith(wrapToken) && trimmed.endsWith(wrapToken);
```
is*** - for method names